hack/push-manifest-list.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+if [[ $# == 0 ]]; then
+    echo "Usage: $0 [manifest-list] [arch0...n]"
```

Comment:
Nice move breaking this non-trivial manifest list logic into something that's generally useful for both the userspace and kernel images. :+1: